transactionoutbox-spring/src/test/java/com/synaos/transactionoutbox/acceptance/EventuallyConsistentControllerTest.java

```diff
@@ -0,0 +1,48 @@
+package com.synaos.transactionoutbox.acceptance;
+
```

Comment:
There is no actual change other than the package name.
- remove unwanted new line or space
- revert changes in code line location for example import order.